pyproject.toml

```
@@ -34,7 +34,7 @@ PyCifRW = "^4.4.6"
 
 [tool.poetry.group.dev.dependencies]
 coverage = {extras = ["toml"], version = "^7.4.1"}
-janus-core = "^0.1.0b2"
+janus-core = "^0.1.0b3"
```

```suggestion
janus-core = "^0.1.0b4"
```
This won't fix it since it's already being used and it's unlikely to ever pick the old version, but the requirement isn't technically accurate anymore.